bedrock/mozorg/templates/mozorg/contribute/thankyou.html

```diff
@@ -43,21 +43,50 @@ <h1 class="page-title">{{ _('Thanks!') }}</h1>
     <div class="container">
       <div class="confirm-body">
 
-        <div class="mozillians-profile">
-          <span class="profile-greeting">{{ _('Hello!') }}</span>
-          <span class="profile-name">{{ _('Your Name') }}</span>
-        </div>
-        <h2 class="section-title">{{ _('In the meantime&hellip;') }}</h2>
-        <p class="section-tagline">{{ _('You can create your Mozillians account now to connect with fellow Mozillians.' ) }}</p>
+      {% if (category == 'coding') or (category == 'testing') or (category == 'helping') or (category == 'translating') %}
+        <div class="mozillians-profile oneandone"></div>
+        <h2 class="section-title">
+          {{ _('In the meantime&hellip;') }}</h2>
```

Comment:
Only because you'll be updating other things, closing tag is in a funky spot here.